src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/footer.html

```diff
@@ -10,9 +10,9 @@
                 </script>
             </div>
             <div class="col-lg-4 my-3 my-lg-0">
-                <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-twitter" aria-hidden="true"></i></a>
-                <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-facebook-f" aria-hidden="true"></i></a>
-                <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-linkedin-in" aria-hidden="true"></i></a>
+                <a class="btn btn-dark btn-social mx-2" href="#!" aria-label="Twitter"><i class="fab fa-twitter" aria-hidden="true"></i></a>
```

Comment:
Change it to the new [X icon](https://fontawesome.com/icons/x-twitter?f=brands&s=solid) and use the label "X (Twitter)". Same for `landingpage.liquid` and the ComingSoon theme too.

Reply:
Okay, but then I will need to open new PRs and wait for them to get approved too.

Reply:
This is OC code, you can just change it.